Description: Track: fix wrong parameter name in removeTag
 ...it is "tag", not "tags". Tag removal works now.
Author: MatÄ›j Laitl <matej@laitl.cz>
Origin: https://github.com/strohel/liblastfm/commit/6b9b3b54a63861376b881ff0e8cf045a5f1b62e7
diff --git a/src/Track.cpp b/src/Track.cpp
index 9ed3b0b..50d95b2 100644
--- a/src/Track.cpp
+++ b/src/Track.cpp
@@ -752,7 +752,7 @@ lastfm::Track::removeTag( const QString& tag ) const
     if (tag.isEmpty())
         return 0;
     QMap<QString, QString> map = params( "removeTag" );
-    map["tags"] = tag;
+    map["tag"] = tag;
     return ws::post(map);
 }
 
